ADOPTION-JOURNEY.md

```diff
@@ -0,0 +1,14 @@
+# Adoption Journey {{version}}
+
+What are the considerations before adopting GitOps? (to avoid disaster or bad taste)
+
+1. Start small
+2. Consider the right skill set at the start of the journey (i.e. Basic understanding of software development lifecycle (SDLC) because GitOps is a software driven target operating model (TOM))
+3. Container orchestration (e.g. Kubernetes (K8s)) has to be on the roadmap - how do we get ready for that (i.e. Start with infrastructure, then network)
+4. Matrix view of crawl / walk / run (i.e. Independent deployability, Rapid application deployment)
+5. Enablement strategy - Git enablement, Configuration Management (e.g. Kustomize) enablement, DevSecOps enablement
+6. Standardisation (i.e. Choose a standard from the options in Argo CD and Flux CD to structure your repositories)
```

Comment:
Flux CD → Flux

Flux is just called "Flux".
https://github.com/open-gitops/documents/pull/47/files#r762415300

```suggestion
6. Standardisation (i.e. Choose a standard from the options in Argo CD and Flux to structure your repositories)
```